### ISSUE TITLE
Server Troubles

### ISSUE THREAD

**Issue by DarceyCayl** [SmartlyDressedGames/Unturned-3.x-Community#306]
Hello,

I have a server on Unturned that's been having some issues since the new update, and was wondering if I could get some help, or some insight on what the problem might be.

Server IP / Port : 64.94.95.130:27265

My server host is Lyhme Hosting, after submitting a support ticket with them, they suggested I submit one here.

Firstly, when trying to connect to my server, I am able find the server in the server browser, I am able to attempt to join, but once the loading bar gets to "Queue Position #1", a moment goes by and I get a white screen followed by an error message saying "You lost your place in the queue".

However, if I try to hard enough, I'll finally be able to join, but just briefly. Once I'm in the game, the messages broadcast from uEssentials has symbols in the text as if there is some sort of error.

I've checked the server logs for any error messages that may show up, and the only thing I can find is when it actually lets me join I can see this : 

```
[5/27/2018 10:22:53 AM] [Info] Connecting: PlayerID: 76561198092772464 Name: WarmCookie Character: [ JEDI ] WarmCookie
[5/27/2018 10:22:54 AM] [Info] Broadcast: [ Connected ] [ JEDI ] WarmCookie
[5/27/2018 10:23:32 AM] [Info] Kicking for inactivity: PlayerID: 76561198092772464 Name: WarmCookie Character: [ JEDI ] WarmCookie
[5/27/2018 10:23:32 AM] [Info] Broadcast: [ Disconnected ] [ JEDI ] WarmCookie
[5/27/2018 10:23:32 AM] [Info] Disconnecting: PlayerID: 76561198092772464 Name: WarmCookie Character: [ JEDI ] WarmCookie
```

The plugins I currently run on the server : 
- Airdrop Manager
- SDPlugins Owner Checker
- uEssentials
- Votifier
- WebsiteCommand
- ZaupFeast

I haven't made any changes to any of the plugins in several weeks, the issue only came up after the server was updated to the newest Steam version.

Any help would be much appreciated!

**Comment by N1GHTPATROL** [comment]
While this isn't RM support, I do notice that the "Kicking for Inactivity" has always been a problem with Unturned, so maybe this is for Nelson. However, if you are having this happen continually then it's probably something with your server setup, which in that case we cannot help with.

**Comment by SDGNelson** [comment]
Considering the strange text symbols my guess would be that it's an issue with Rocket plugins needing an update to the latest RocketMod version, as several plugins needed adjusting to new chat features.

If the problem persists I'll investigate further. The inactivity kick is to prevent troll/fake clients from sitting at the front of the queue holding everyone up, however plugins shouldn't be able to mess with that.

**Comment by DarceyCayl** [comment]
Thanks for the feedback guys. I'm not totally sure what the issue was, but I was able to fix the server! I removed the plugins one by one to make sure they weren't the problem, then I removed RocketMod all together, and yet the server was still acting up. So then I wiped the server data to see if something may have corrupted, and still no changes. 

Lastly, I deleted the config.json file for the server to restore it to default and badda bing! No clue what was set up differently that caused the problem, but it's fixed now!

Thanks again!
